packages/dapp-svelte-wallet/ui/src/Transaction.svelte

```diff
@@ -114,7 +114,7 @@
     {/if}
     {#if status === 'proposed'}
     <div class="flex flex-row">
-    <Chip on:click={() => E(walletP).acceptOffer(offerId).then(showOutcome)}
+    <Chip on:click={() => E(walletP).acceptOffer(offerId)}
```

Comment:
Is there some other pathway to displaying the success/failure of `acceptOffer`? (I have no idea how this works, just surprised to see a removal without an addition elsewhere)

Reply:
It would be good to display a message on rejection.  I'll add that.

Reply:
(Success doesn't need a display, since now the Dapps do it themselves.)